lib/presentation/shared/elevated_buttons.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+abstract class SBElevatedButton extends StatelessWidget {
+  final VoidCallback? onTap;
+
+  const SBElevatedButton({Key? key, this.onTap}) : super(key: key);
+
+  Widget childWidget();
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: onTap,
+      child: childWidget(),
+    );
+  }
+}
+
+class SBPrimaryElevatedButton extends SBElevatedButton {
```

Comment:
I think it is unnecessary to create button class, since shape and even elevation is coming from theme.

just define 3 themes

primaryFilledButtonTheme
dangeFilledButtonTheme
inactiveFilledButtonTheme
![Screenshot 2023-02-02 at 9 35 52 AM](https://user-images.githubusercontent.com/35165481/216232607-88da297d-20f7-4643-85b4-f77d8dd6e4e6.png)

![Screenshot 2023-02-02 at 9 36 13 AM](https://user-images.githubusercontent.com/35165481/216232666-3f1fb18e-f0ce-4ed1-a25a-07cb3629a366.png)